src/Data/CSV/Conduit/Parser/Text.hs

```diff
@@ -58,15 +58,17 @@ csv s = do
 row :: CSVSettings -> Parser (Maybe (Row Text))
 row csvs = csvrow csvs <|> badrow
 
+csvEndOfLine :: Parser ()
+csvEndOfLine = (char '\n' >> return ()) <|> (string (T.pack "\r\n") >> return ()) <|> (char '\r' >> return ())
```

Comment:
Copied from https://hackage.haskell.org/package/attoparsec-0.13.0.1/docs/src/Data-Attoparsec-Text-Internal.html
and modified to handle '\r'.